src/prof_sys.c

```diff
@@ -97,13 +97,17 @@ prof_backtrace_impl(void **vec, unsigned *len, unsigned max_len) {
 
 	cassert(config_prof);
 	assert(vec != NULL);
-	assert(max_len == PROF_BT_MAX);
+	assert(max_len <= PROF_BT_MAX);
```

Comment:
PROF_BT_MAX will only be meaningful with PROF_GCC now, is that right? If so let's rename it to make it clear, and tweak the logic too.

Reply:
Modified to PROF_BT_MAX_PROF_GCC